v1/proto/gribi_aft/enums/enums.pb.go

```diff
@@ -10,7 +10,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.27.1
-// 	protoc        v3.19.3
+// 	protoc        v3.19.2
```

Comment:
I don't know if this is expected given #27 made it 3.19.3

Reply:
Oops - some version drift between my desktop machine and VM. Fixed now.